pulseapi/entries/views.py

```diff
@@ -502,3 +508,19 @@ def post(self, request, *args, **kwargs):
                 "post validation failed - {}".format(validation_result),
                 status=status.HTTP_400_BAD_REQUEST
             )
+
+
+class ProjectEntriesListView(EntriesListView):
```

Comment:
hmm, have you tested this? I'm not sure if the `get_serializer_class` method in the parent class will override the `serializer_class` here because I think it does that if it's in the same class.

Reply:
we don't want it to, though - this is active on a different endpoint URL, so we always want this view to use the project serializer. Were you thinking about the v1 vs. v2 distinction? (do we need that if this only exists as of v2, given that v1 is basically "locked in the past"?)

Reply:
both actually (I didn't think about versioning so glad you brought that up). And yeah you're right, we don't want it to override, but just checking that it actually doesn't override the `serializer_class` property because in:
```py
class SomeView(ListAPIView):
    serializer_class = A

    def get_serializer_class(self):
        return B
```
The serializer used is `B` because it always calls `get_serializer_class` and `ListAPIView` has a default implementation for `get_serializer_class` that returns the value of the property `serializer_class`.

So, when inheriting:
```py
class SomeOtherView(SomeView):
    serializer_class = C
```
since it always calls `get_serializer_class`, I would think it would go up the inheritance chain (`SomeOtherView`-->`SomeView`-->`ListAPIView`) and find an implementation of `get_serializer_class` in `SomeView` and execute that...so the serializer used would be `B` and not `C`.

I'm not 100% sure about that last bit which is why I'm asking if you tested to see if it works.